lib/Models/ArcGisFeatureServerCatalogGroup.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+/*global require*/
+var URI = require('urijs');
+
+var clone = require('terriajs-cesium/Source/Core/clone');
+var defined = require('terriajs-cesium/Source/Core/defined');
+var defineProperties = require('terriajs-cesium/Source/Core/defineProperties');
+var freezeObject = require('terriajs-cesium/Source/Core/freezeObject');
+var knockout = require('terriajs-cesium/Source/ThirdParty/knockout');
+var loadJson = require('terriajs-cesium/Source/Core/loadJson');
+var when = require('terriajs-cesium/Source/ThirdParty/when');
+
+var TerriaError = require('../Core/TerriaError');
+var CatalogGroup = require('./CatalogGroup');
+var inherit = require('../Core/inherit');
+var proxyCatalogItemUrl = require('./proxyCatalogItemUrl');
+var replaceUnderscores = require('../Core/replaceUnderscores');
+var ArcGisFeatureServerCatalogItem = require('./ArcGisFeatureServerCatalogItem');
+
+/**
+ * A {@link CatalogGroup} representing a collection of layers from an ArcGIS Feature Service.
+ * Eg. https://sampleserver6.arcgisonline.com/arcgis/rest/services/Wildfire/FeatureServer
+ *
+ * @alias ArcGisFeatureServerCatalogGroup
```

Comment:
It used to be required in order for jsdoc to work right.  But that was about a hundred years ago, I have no idea if it still is.